test/distributed/tensor/test_attention.py

```diff
@@ -86,6 +89,10 @@ def _test_ring_attention_sdpa(
         rotater: _RotateMethod,
         test_forward_only: bool,
     ) -> None:
+        # TODO: SDPBackend.CUDNN_ATTENTION does not support backward so far
+        if not test_forward_only and backend == SDPBackend.CUDNN_ATTENTION:
```

Comment:
Just to confirm this is specific to the Dtensor impl not the op itself - since that does support bwd

Reply:
yes, the next step is to add backward DTensor impl.